apps/slackbot/README.md

```diff
@@ -24,39 +24,56 @@ This repository contains a chatbot implementation using Flask and Slack. The cha
     https://medium.com/developer-student-clubs-tiet/how-to-build-your-first-slack-bot-in-2020-with-python-flask-using-the-slack-events-api-4b20ae7b4f86
```

Comment:
What about we add the steps from this blog to this project rather than directing people out of the repo

Reply:
I changed it to the Slackbot tutorial in this repo instead.